shared/actions/settings.tsx

```diff
@@ -393,6 +395,23 @@ const loadSettings = () =>
     })
   })
 
+const computeBadgesOnLoadedSettings = (state, action: SettingsGen.LoadedSettingsPayload) => {
+  const emailCount = (action.payload.emails || I.Map()).reduce(
+    (count, row) => (row.isVerified ? count : count + 1),
+    0
+  )
+  const phoneCount = (action.payload.phones || I.Map()).reduce(
+    (count, row) => (row.verified ? count : count + 1),
+    0
+  )
+
+  return NotificationsGen.createSetBadgeCounts({
+    counts: I.Map({
+      [Tabs.settingsTab as Tabs.Tab]: emailCount + phoneCount,
+    }) as I.Map<Tabs.Tab, number>,
+  })
+}
```

Comment:
Actually one more suggestion to evaluate: can you do this inside `loadSettings` instead? You can `SettingsGen.createLoadedSettings()` and `NotificationsGen.createSetBadgeCounts()` in parallel.

Reply:
How do I return more than one action?